src/App.css

```diff
@@ -23,8 +23,48 @@ div {
   color: black;
   display: flex;
   flex-wrap: wrap;
-  max-height: 100%;
-  max-width: 100%;
+  /* max-height: 100%;
+  max-width: 100%; */
```

Comment:
If you are not going to use this styling it should be removed rather than commented out.